### DISCUSSION TITLE
Setup disabled

### DISCUSSION THREAD

**Discussion by RedICEY** [Pryaxis/TShock#3047] [Q&A]
I can't use any admin thing as the code for setting up isn't showing anymore and when I do /setup it says that setup has been disabled and has been logged, I've been locked out of admin and cant do anything

**Comment by sgkoishi** [comment]
check the pinned guide Pryaxis/TShock#2325

**Comment by zyrafaq** [comment]
Manually set someone as admin, first create an admin group or user `superadmin` and type `/user group <user account name> <group name>`. Please note that `superadmin` is not reccommended and instead create a group called `owner`. Hope this helps
